com.gamefuse.sdk/Runtime/GameFuseStoreItem.cs

```diff
@@ -24,31 +24,12 @@ public class GameFuseStoreItem : MonoBehaviour
         #endregion
 
         #region instance getters
-        public string GetName()
-        {
-            return name;
-        }
-        public string GetCategory()
-        {
-            return category;
-        }
-        public string GetDescription()
-        {
-            return description;
-        }
-        public int GetCost()
-        {
-            return cost;
-        }
-        public int GetId()
-        {
-            return id;
-        }
-        public string GetIconUrl()
-        {
-            return icon_url;
-        }
-
+        public string GetName() => name;
```

Comment:
Cool syntax

Reply:
yeah man I just learned that syntax recently, pretty cool.